packages/app-page-builder/src/editor/plugins/elementSettings/action/ActionSettings.tsx

```diff
@@ -39,9 +39,14 @@ const ActionSettingsComponent: React.FC<ActionSettingsPropsType> = ({
 }) => {
     const updateElement = useUpdateElement();
 
-    const { actionType } = element.data?.action || { actionType: "link" };
+    const { actionType } = element.data?.action || {};
 
     const updateSettings: FormOnSubmit = data => {
+        // Skip update if nothing is changed.
+        if (data.actionType === actionType) {
```

Comment:
Let's double check this. It looks like now I can't save button action settings at all. 
![image](https://github.com/webiny/webiny-js/assets/5121148/6727993d-209f-4746-b471-6c95cc599f2b)